gemd/units/impl.py

```diff
@@ -307,7 +307,7 @@ def _format_clean(unit, registry, **options):
     """
     try:  # Informal route changed in 0.22
         from pint.formatting import _FORMATTERS
-        formatter = _FORMATTERS["D"]
+        formatter = _FORMATTERS["D"]  # pragma: no cover
```

Comment:
This is unreachable with the current library limits, but is reachable for older versions of Pint.  Kept to avoid being "breaking" without a major version bump.